src/logger/loggerApi.c

```diff
@@ -267,6 +269,29 @@ int api_getCapabilities(struct Serial *serial, const jsmntok_t *json)
         return API_SUCCESS_NO_RETURN;
 }
 
+static void get_wifi_status(struct Serial* serial, const bool more)
+{
+        const struct wifi_ap_cfg* ap_cfg = esp8266_drv_get_ap_config();
+        const struct wifi_client_cfg* clt_cfg = esp8266_drv_get_client_config();
+
+        const bool ap_active = ap_cfg && ap_cfg->active;
+        const bool client_active = clt_cfg && clt_cfg->active;
+        const bool client_connected = esp8266_drv_client_connected();
+
+        json_objStartString(serial, "wifi");
+
+        json_objStartString(serial, "ap");
+        json_bool(serial, "active", ap_active, false);
```

Comment:
After discussing with @ryandoherty in Slack we are going to leave things as is until we know more about what is helpful for diagnostic info.  Then we can add new metrics as needed.